google-style.yml

```diff
@@ -35,13 +35,18 @@ Metrics/PerceivedComplexity:
   Max: 10
 Naming/VariableNumber:
   Enabled: false
+Style/BlockDelimiters:
+  EnforcedStyle: braces_for_chaining
```

Comment:
Just curious, what are the reasons for the `braces_for_chaining` style?

Reply:
Changes `datastore.query("Visit").order("timestamp", :desc).limit 10` to `datastore.query("Visit").order("timestamp", :desc).limit(10)`
I'm not super dogmatic about it one way or the other, but I prefer the second. Do you prefer the first?

Reply:
Are you sure you have the right rule? See https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/BlockDelimiters:

```rb
EnforcedStyle: braces_for_chaining
# bad
words.each do |word|
  word.flip.flop
end.join("-")

# good
words.each { |word|
  word.flip.flop
}.join("-")
```

Reply:
I prefer `line_count_based` for `BlockDelimiters`. Another approach I've seen is to use curly braces when you care about the return value, and do/end when you don't. I never liked that style myself, but that seems to be what `braces_for_chaining` is specifying.

Reply:
@quartzmo yeah, got them mixed up. I actually don't really have feelings on this one, other than keeping line_count_based enabled

Reply:
@TheRoyalTnetennba Can you remove `EnforcedStyle: braces_for_chaining` then?

Reply:
We can. The discussion for ruby-docs-samples seemed to favor `braces_for_chaining`. I'm close to ambivalent. Do you have a preference?

Reply:
I would prefer `line_count_based` and to have ruby-docs-samples override this rule in their own configuration.